2. flow_control/Q17.c

```diff
@@ -1 +1,30 @@
 // Write a program to read an array of n integers and find the maximum and minimum values in the array using a for loop.
+#include<stdio.h>
+int main(){
+    int array[10],n,i,maximum,minimum;
```

Comment:
use array[n] , if we don't use variable array then there is no use of taking  array size as a input from user

Reply:
#include<stdio.h>
int main(){
    int n,i,maximum,minimum;
    printf("Number of elements in the array = ");
    scanf("%d", &n);
    int array[n];